src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageFeed.cs

```diff
@@ -257,6 +264,8 @@ public async Task<SearchResult<IPackageSearchMetadata>> RefreshSearchAsync(Refre
                         telemetryState.PageIndex,
                         aggregated.Items?.Count ?? 0,
                         telemetryState.Duration.Elapsed,
+                        timings,
+                        timeAggregation.Elapsed,
```

Comment:
can `timeAggregation` ever be null by this time?

Reply:
No, neither will timings. This can be cleaner.